flow/project.py

```diff
@@ -588,8 +588,16 @@ def _determine_entrypoint(self, entrypoint, directives, job):
         else:
             return "{} {}".format(entrypoint['executable'], entrypoint['path']).lstrip()
 
-    def _submit_cmd(self, entrypoint, ignore_conditions, job=None):
-        entrypoint = self._determine_entrypoint(entrypoint, self.directives, job)
+    def resolve_directives(self, name, defaults):
+        if len(self.directives) != 0:
+            directives = deepcopy(self.directives)
+            directives.update(self.operation_directives.get(name, dict()))
+        else:
+            directives = deepcopy(self.operation_directives.get(name, default.get(name, dict())))
```

Comment:
```suggestion
            directives = deepcopy(self.operation_directives.get(name, defaults.get(name, dict())))
```

Reply:
Noticed that, too. :+1:

Reply:
Thanks, I was trying to fix a bug @bdice noticed yesterday, and didn't run tests before pushing. I must have missed it.